pipeline.yml

```diff
@@ -0,0 +1,50 @@
+jobs:
+- name: deploy-drupal-7
+  plan:
+  - aggregate: *plan-gets
+  - put: deploy-drupal-7
+    params:
+      path: cf-ex-drupal
+      manifest: cf-ex-drupal/manifest.yml
+  on_failure:
+    put: slack
+    params:
+      <<: *slack-params
```

Comment:
Same as above--this reference need to be defined for this to be valid yaml. Also, I used references in the sample apps pipeline to avoid duplicating boilerplate config, but since this pipeline only deploys one app, it would probably be clearer not to use them at all.